web/impact/impact/tests/test_user_list_view.py

```diff
@@ -3,11 +3,18 @@
 
 from django.urls import reverse
 from django.contrib.auth import get_user_model
-
+from impact.models import (
+    EntrepreneurProfile,
+    ExpertProfile,
+    MemberProfile
+)
+import pytz
+from impact.tests.factories import StartupTeamMemberFactory
 from impact.tests.contexts import UserContext
 from impact.tests.api_test_case import APITestCase
+import simplejson as json
 from impact.v1.views.user_list_view import EMAIL_EXISTS_ERROR
-
+import datetime
```

Comment:
switching to this and removing the pytz.utc didnt seem to get rid of the warning messages when running tests.